src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1270,6 +1270,8 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 			gamerule.description = modElement.getName();
 			gamerule.category = getRandomString(random, Arrays.asList("PLAYER", "UPDATES", "CHAT", "DROPS", "MISC", "MOBS", "SPAWNING"));
 			gamerule.type = new String[] { "Number", "Logic", "Number", "Logic" }[valueIndex];
+			gamerule.defaultValueLogic = _true;
+			gamerule.defaultValueNumber = -45;

```

Comment:
Why is this set to -45?

Reply:
It is an arbitrary non-default value to test UI saving capability.